crates/uv-extract/src/sync.rs

```diff
@@ -17,7 +17,7 @@ pub fn unzip<R: Send + std::io::Read + std::io::Seek + HasLength>(
     let archive = ZipArchive::new(CloneableSeekableReader::new(reader))?;
     let directories = Mutex::new(FxHashSet::default());
```

Comment:
We could also probably use a `dashmap::DashSet` here, if you're interested in making that change as well.